container_pipeline/workers/linter.py

```diff
@@ -81,15 +81,37 @@ def lint(self):
             out, err = run_cmd_out_err(command)
             if err == "":
                 response = self.handle_lint_success(out)
+                self.logger.info(
```

Comment:
This logger statement needs to come before we try to do `handle_lint_success` because that method tries to create an OpenShift project. Doesn't make sense to log trying to do something after trying it.

Reply:
Done